client/src/main/java/io/pravega/client/stream/StreamCut.java

```diff
@@ -63,7 +63,8 @@ private Object readResolve() {
     
     /**
      * Serializes the cut to a compact byte array.
-     * @return A serialized version of this streamcut.
+     *
+     * @return a compact byte array as a ByteBuffer
```

Comment:
What was wrong before?

Reply:
Nothing. I think it was just a matter of picking one out of a merge where two of us fixed a return warning. I'll change it back.